api/src/main/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGenerator.java

```diff
@@ -41,20 +41,22 @@
 	 * @should throw exception if config is not for registrationapp
 	 * @should return empty Extension list if app config has no sections
 	 */
-	public static List<Extension> generate(AppDescriptor app) throws IOException {
+	public static List<Extension> generate(AppDescriptor app) throws  IllegalArgumentException{
```

Comment:
👍for choosing `IllegalArgumentException `.

However leave a space before the opening bracket: ` IllegalArgumentException{` 
→ `IllegalArgumentException {`.

Sorry about those, it looks like this module is not yet hooked to a code formatter leading to this waste of time... :-(

Reply:
I corrected these typos. Thank you @mks-d .